app/templates/content/privacy-notice.html

```diff
@@ -0,0 +1,167 @@
+{% extends "toolkit/layouts/_base_page.html" %}
+
+{% block page_title %}How your data is used (‘Privacy Notice’) - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": url_for('.index'),
+          "label": "Digital Marketplace"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  <div class="help-page grid-row legal-content">
+    <div class="column-two-thirds">
+
+      
+      <header class="page-heading">
+        <h1>How your data is used (‘Privacy Notice’)</h1>
+      </header>
+      <p>The Digital Marketplace is built and run by the Government Digital Service (GDS).</p>
+      <p>We collect your personal data as part of our service. We use it when you create user accounts on our platform, in order to buy or sell digital services. This is fully compliant with UK and EU law.</p>
+
+
+      <div class="dmspeak">
+        <h2 class="heading-xmedium">How and why your data is collected</h2>
+      </div>
+
+      <h3>What personal data we collect</h3>
+      <div class="check-list">
+        <p class="lead">We collect several points of personal data from our users. These are:</p>
+        <ul class="list-bullet">
+          <li>email addresses and contact details (phone number and address)</li>
+          <li>company and company contact information (suppliers only)</li>
+        </ul>
+      </div>
+
+      <h3>Your email address and contact details</h3>
+      <div class="check-list">
+        <p class="lead">When you give us your email we will only use it so that you can:</p>
+        <ul class="list-bullet">
+          <li>log in securely</li>
+          <li>be contacted about a particular contract, application or brief</li>
+          <li>receive updates from a mailing list you’ve subscribed to (for example if you’ve asked to be notified when a new framework opens)</li>
+        </ul>
+      </div>
+
+      <h3>Your company information</h3>
+      <div class="check-list">
+        <p>When suppliers register for the Digital Marketplace, they need to provide us with several pieces of data, including their company contact information.</p>
+        <p class="lead">We use that to:</p>
+        <ul class="list-bullet">
+          <li>inform buyers who they can get in touch with at your company</li>
+          <li>register a company on a framework to sell services</li>
+          <li>update suppliers on the status of their applications to frameworks or briefs</li>
+        </ul>
+      </div>
+      <p>You will always be told if the personal data you provide is going to be displayed on the Digital Marketplace.</p>
+
+      <h3>Cookies and analytics</h3>
+      <p>You can read our <a href="/cookies">full cookie policy</a> and to find out how we use cookies to track user journeys using Google Analytics.</p>
```

Comment:
This should use the url_for method to get the cookie external path.

Reply:
👍 Good shout